app/components/DR/ActivityIndicator.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ActivityIndicator } from 'react-native';
+
+const activityIndicatorLoadingView = () => {
+  //making a view to show to while loading the webpage
+  return (
+    <ActivityIndicator
+      color='#009688'
```

Comment:
Instead of this color you should use the one in Colors.BLUE_RIBBON, because the one you use is not in the app color palette.

Reply:
Yeah